SpatialGDK/Source/SpatialGDKEditor/Private/SpatialTestSettings.cpp

```diff
@@ -49,15 +49,25 @@ void FSpatialTestSettings::Override(const FString& MapName)
 
 	if (ASpatialWorldSettings* SpatialWorldSettings = Cast<ASpatialWorldSettings>(World->GetWorldSettings()))
 	{
-		FString GroupOverridesFilename = FPaths::ProjectDir() + SpatialWorldSettings->SettingsOverride.FilePath;
-		if (FPaths::FileExists(GroupOverridesFilename))
+		// First we check if the world settings override path exists in itself.
+		// If it doesn't we see if it is a relative path from the project.
+		// If not we give up...
+
+		if (FPaths::FileExists(SpatialWorldSettings->SettingsOverride.FilePath))
```

Comment:
Should we be allowing absolute paths? is this portable between machines?
Or is this still relative to something?